src/app/models/redirect-data.ts

```diff
@@ -0,0 +1,6 @@
+export class RedirectData {
+  message: string;
+  secMessage: string;
+  ctaLink: string;
+  ctaLabel: string;
```

Comment:
Could these properties be renamed so that they're a little more self-explanatory?

Reply:
Do you have any suggestion?